public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
This is using the font from google apis instead of the ttf file you uploaded with this pr, you must decide if you want use the api or the file

Reply:
I'll use this link rather

Reply:
Rather go for the ttf file, its better if you ever find yourself doing offline dev, use %PUBLIC_URL% and the font file

Reply:
Ok here's what I did to use the file. 

1) created 'font' file under src called 'Montserrat'
2) imported in index.js
3)added in index.css
4)added className in App.css 'font-face-gm'
5)used className in profileHeader.js and profileBody.js

should be working fine